ppx/browser/ppx_deriving_json_runtime.ml

```diff
@@ -68,6 +71,14 @@ module Of_json = struct
       else of_json_error "expected an integer"
     else of_json_error "expected an integer"
 
+  let int64_of_json (json : t) : int64 =
+    if Js.typeof json = "string" then
+      let v = (Obj.magic json : string) in
+      match Int64.of_string_opt v with
+      | Some v -> v
+      | None -> of_json_error "expected an int64"
```

Comment:
Maybe it should say (in all other places too) something like “int64 as string”

Reply:
added in https://github.com/melange-community/melange-json/pull/33/commits/70673c14228b3f77ef544c395eadffc8fdd6f71e.